particle_list.yml

```diff
@@ -1,821 +1,780 @@
-      BaryonNumber: 0
-
   e-:
     PID: -11
-    Mass: { Value: 0.0005109989461 }
+    Mass: 0.0005109989461
     QuantumNumbers:
       Spin: 0.5
       Charge: -1
-      Parity: +1
+      Parity: 1
       ElectronLN: 1
-      MuonLN: 0
-      TauLN: 0
-      BaryonNumber: 0
-
   mu+:
     PID: 13
-    Mass: { Value: 0.1056583745 }
+    Mass: 0.1056583745
     QuantumNumbers:
       Spin: 0.5
       Charge: 1
       Parity: -1
-      ElectronLN: 0
       MuonLN: -1
-      TauLN: 0
-      BaryonNumber: 0
-
   mu-:
     PID: -13
-    Mass: { Value: 0.1056583745 }
+    Mass: 0.1056583745
     QuantumNumbers:
       Spin: 0.5
       Charge: -1
-      Parity: +1
-      ElectronLN: 0
+      Parity: 1
       MuonLN: 1
-      TauLN: 0
-      BaryonNumber: 0
-
   tau+:
     PID: 15
-    Mass: { Value: 1.77686 }
+    Mass: 1.77686
     QuantumNumbers:
       Spin: 0.5
       Charge: 1
       Parity: -1
-      ElectronLN: 0
-      MuonLN: 0
       TauLN: -1
-      BaryonNumber: 0
-
   tau-:
     PID: -15
-    Mass: { Value: 1.77686 }
+    Mass: 1.77686
     QuantumNumbers:
       Spin: 0.5
       Charge: -1
-      Parity: +1
-      ElectronLN: 0
-      MuonLN: 0
+      Parity: 1
       TauLN: 1
-      BaryonNumber: 0
-
   ve:
     PID: 12
-    Mass: { Value: 1e-9 }
+    Mass:
+      Value: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: 1
       ElectronLN: 1
-      MuonLN: 0
-      TauLN: 0
-      BaryonNumber: 0
-
   vebar:
     PID: -12
-    Mass: { Value: 1e-9 }
+    Mass:
+      Value: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: -1
       ElectronLN: -1
-      MuonLN: 0
-      TauLN: 0
-      BaryonNumber: 0
-
   vmu:
     PID: 14
-    Mass: { Value: 1e-9 }
+    Mass:
+      Value: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: 1
-      ElectronLN: 0
       MuonLN: 1
-      TauLN: 0
-      BaryonNumber: 0
-
   vmubar:
     PID: -14
-    Mass: { Value: 1e-9 }
+    Mass:
+      Value: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: -1
-      ElectronLN: 0
       MuonLN: -1
-      TauLN: 0
-      BaryonNumber: 0
-
   vtau:
     PID: 16
-    Mass: { Value: 1e-9 }
+    Mass:
+      Value: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: 1
-      ElectronLN: 0
-      MuonLN: 0
       TauLN: 1
-      BaryonNumber: 0
-
   vtaubar:
     PID: -16
-    Mass: { Value: 1e-9 }
+    Mass: 1e-9
     QuantumNumbers:
       Spin: 0.5
       Charge: 0
       Parity: -1
-      ElectronLN: 0
-      MuonLN: 0
       TauLN: -1
-      BaryonNumber: 0
-
   pi0:
     PID: 111
-    Mass: { Value: 0.1349766, Error: 0.000006 }
+    Mass:
+      Value: 0.1349766
+      Error: 0.000006
     QuantumNumbers:
       Spin: 0
       Charge: 0
       Parity: -1
       Cparity: 1
       Gparity: -1
-      IsoSpin: { Value: 1, Projection: 0 }
-      BaryonNumber: 0
-      Charm: 0
-      Strangeness: 0
-
+      IsoSpin:
+        Value: 1
```

Comment:
Magnitude

Reply:
I agree that Magnitude would be a better name, but I've tried not to change the expected structure of the YAML file (i.e. names of the keys and such). So, strictly speaking, I would separate this suggestion into a new issue, so that this PR only pertains to the introduction of a `ParticleDatabase` that can read from the existing `particle_list.yml` and `particle_list.xml` files.